--- pdbsql-0.1-samba_3.0.23/Makefile.in~	2006-06-14 23:18:54.000000000 +0300
+++ pdbsql-0.1-samba_3.0.23/Makefile.in	2006-07-25 11:19:04.200956844 +0300
@@ -11,7 +11,7 @@
 PGSQL_LIBS = @PGSQL_LIBS@
 SOURCEDIR = @SOURCEDIR@
 LIBDIR = @LIBDIR@
-CFLAGS = -Wall @CFLAGS@ -I$(SOURCEDIR) -I$(SOURCEDIR)/include -I$(SOURCEDIR)/ubiqx -I$(SOURCEDIR)/popt -I$(SOURCEDIR)/lib/replace -I$(SOURCEDIR)/tdb/include -I$(SOURCEDIR)/librpc
+CFLAGS = -Wall @CFLAGS@ -I$(SOURCEDIR) -I$(SOURCEDIR)/include -I$(SOURCEDIR)/ubiqx -I$(SOURCEDIR)/popt -I$(SOURCEDIR)/lib/replace -I$(SOURCEDIR)/tdb/include -I$(SOURCEDIR)/librpc -I$(SOURCEDIR)/smbwrapper -I$(SOURCEDIR)/lib/tdb/include -I$(SOURCEDIR)/lib/talloc
 
 all: $(addsuffix .$(SHLIBEXT), $(modules))
 
--- pdbsql-0.1-samba_3.0.23/configure.in	2006-07-25 11:23:19.876700234 +0300
+++ pdbsql-0.1-samba_3.0.23/configure.in	2006-07-25 11:52:43.626364490 +0300
@@ -36,15 +36,14 @@
  	SOURCEDIR="$SOURCEDIR/source"
 else if test -f "$SOURCEDIR/dynconfig.c"; then
 	SOURCEDIR="$SOURCEDIR"
+elif test -f "$SOURCEDIR/includes.h"; then
+	SOURCEDIR="$SOURCEDIR"
      else 
 	AC_MSG_ERROR([Unable to find Samba sources in $SOURCEDIR. Please specify correct value to --with-samba-dir=])
   fi
 fi
 
-if test -z "$LIBDIR"; then
-	prefix=`grep "^prefix=" $SOURCEDIR/Makefile | cut -d = -f 2`
-	LIBDIR="$prefix/lib"
-fi
+LIBDIR="$libdir"
 
 AC_MSG_RESULT([$SOURCEDIR])
 
