--- pdbsql-0.1-samba_3.0.23/Makefile.in~	2006-06-14 23:18:54.000000000 +0300
+++ pdbsql-0.1-samba_3.0.23/Makefile.in	2006-07-25 11:19:04.200956844 +0300
@@ -11,7 +11,7 @@
 PGSQL_LIBS = @PGSQL_LIBS@
 SOURCEDIR = @SOURCEDIR@
 LIBDIR = @LIBDIR@
-CFLAGS = -Wall @CFLAGS@ -I$(SOURCEDIR) -I$(SOURCEDIR)/include -I$(SOURCEDIR)/ubiqx -I$(SOURCEDIR)/popt
+CFLAGS = -Wall @CFLAGS@ -I$(SOURCEDIR) -I$(SOURCEDIR)/include -I$(SOURCEDIR)/ubiqx -I$(SOURCEDIR)/popt -I$(SOURCEDIR)/smbwrapper
 
 all: $(addsuffix .$(SHLIBEXT), $(modules))
 
--- pdbsql-0.1-samba_3.0.23/configure.in~	2006-06-14 23:18:54.000000000 +0300
+++ pdbsql-0.1-samba_3.0.23/configure.in	2006-07-25 11:23:19.876700234 +0300
@@ -41,10 +41,7 @@
   fi
 fi
 
-if test -z "$LIBDIR"; then
-	prefix=`grep "^prefix=" $SOURCEDIR/Makefile | cut -d = -f 2`
-	LIBDIR="$prefix/lib"
-fi
+LIBDIR="$libdir"
 
 AC_MSG_RESULT([$SOURCEDIR])
 
